tests/fixtures_valid.js

```diff
@@ -5,15 +5,15 @@ const path = require('path');
 const util = require('util');
 const colors = require('colors');
 
-const fixturePath = path.join(__dirname, '..', 'fixtures');
-const schemas = require(path.join(fixturePath, 'schema'));
-const checkFixture = require(path.join(__dirname, 'fixture_valid')).checkFixture;
+const schemas = require('../fixtures/schema.js');
+const checkFixture = require('./fixture_valid.js').checkFixture;
 
 let usedShortNames = [];
 
 let promises = [];
 
 // search fixture files
+const fixturePath = path.join(__dirname, '..', 'fixtures');
```

Comment:
Actually, you're right. It makes no sense to use it for paths which we later concatenate with `path.join()` anyway.